spec/support/shared_examples/email_header_examples.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'email with inactive white labelling' do |mail|
+  it 'always displays the OFN header with logo' do
+    expect(public_send(mail).body).to include(ContentConfig.url_for(:logo))
+  end
+end
+
+RSpec.shared_examples 'non-customer facing email with active white labelling' do |mail|
+  context 'when hide OFN navigation is enabled for the distributor of the order' do
+    before do
+      allow(order.distributor).to receive(:hide_ofn_navigation).and_return(true)
```

Comment:
You copied the same logic used before but using `allow` is usually only a last resort because it can hide some problems. It's better to set a variable the same way the real code would:
```suggestion
      order.distributor.hide_ofn_navigation = true
```

Reply:
I thought that was a good idea, so I committed it :)